go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/codacy/codacy-engine-golang-seed
+
+go 1.13
+
+require (
+	github.com/360EntSecGroup-Skylar/goreporter v0.0.0-20180902115603-df1b20f7c5d0 // indirect
+	github.com/facebookgo/inject v0.0.0-20180706035515-f23751cae28b // indirect
+	github.com/facebookgo/structtag v0.0.0-20150214074306-217e25fb9691 // indirect
+	github.com/fatih/color v1.8.0 // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
+	github.com/josemiguelmelo/gofile v0.0.0-20200102193424-d0d563814b61
```

Comment:
yep, I would ask you to depend only on standard library and check if there is any upstream library that does what you want, worst case scenario please copy the single source file of your library here.

Reply:
Added a utility method to parse the json file and replace my library call